standalone.js

```diff
@@ -0,0 +1,47 @@
+const { setQueues, router } = require('./dist/index')
+const Queue = require('bull')
+const redis = require('redis')
+const app = require('express')()
+
+const redisOptions = {
+  redis: {
+    port: process.env.REDIS_PORT || 6379,
+    host: process.env.REDIS_HOST || 'localhost',
+    password: process.env.REDIS_PASSWORD || '',
```

Comment:
This doesn't work if no password is set, it should be
```suggestion
    password: process.env.REDIS_PASSWORD || undefined,
```